qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/modules/kubernetes/services/prefect/main.tf

```diff
@@ -3,6 +3,9 @@ resource "helm_release" "prefect" {
   namespace = var.namespace
   chart     = "${path.module}/chart"
```

Comment:
Remove the helm chart. See this forthe chart name and url https://github.com/PrefectHQ/server/tree/master/helm/prefect-server#installing-released-versions.

Reply:
Sure, I've might forgotten that one, tks

Reply:
This one will be adressed in another PR, see this issue for ref. https://github.com/Quansight/qhub/issues/819